test/tests/1d_dc/NonlocalPotentialBCWithSchottky.i

```diff
@@ -498,36 +498,6 @@ area = 5.02e-7 # Formerly 3.14e-6
 
 [BCs]
   ## Potential boundary conditions ##
```

Comment:
```suggestion
  ## Potential boundary conditions ##
  #       [potential_left]
  #               type = NeumannCircuitVoltageMoles_KV
  #               variable = potential
  #               boundary = left
  #               function = potential_bc_func
  #               ions = Arp
  #               data_provider = data_provider
  #               electrons = em
  #               electron_energy = mean_en
  #               r = 0
  #               position_units = ${dom0Scale}
  #       []
  
```

`NeumannCircuitVoltageMoles_KV` isn't being removed here, so let's bring this half of the comments back.